md/rest-api-overview.md

```diff
@@ -161,7 +161,6 @@ For a GET method that retrieves more than one instance of a resource, you can sp
 * p: index of the page to display
 * c: maximum number of elements to retrieve
 * o: order of presentation of values in response: must be either `attributeName ASC` or `attributeName DESC`. The final order parameter value must be URL encoded. 
```

Comment:
Maybe you should precise that possible values for attributeName may be different for every API